src/WebJobs.Script.WebHost/Diagnostics/MetricsEventManager.cs

```diff
@@ -428,33 +427,27 @@ public void Dispose()
             internal void FunctionStarted(FunctionStartedEvent startedEvent)
             {
                 _totalExecutionCount++;
+                Interlocked.Increment(ref _activeFunctionCount);
 
                 var monitoringEvent = new FunctionMetrics(startedEvent.FunctionMetadata.Name, ExecutionStage.Started, 0);
                 _functionMetricsQueue.Enqueue(monitoringEvent);
 
-                var key = GetDictionaryKey(startedEvent.FunctionMetadata.Name, startedEvent.InvocationId);
-                var triggerType = startedEvent.FunctionMetadata.Trigger?.Type;
-                var value = new RunningFunctionInfo(startedEvent.FunctionMetadata.Name, startedEvent.InvocationId, startedEvent.Timestamp, startedEvent.Success, triggerType);
```

Comment:
Removing this type completely. It was just a copy of data we already have in the started event, so we just track those instances directly.